### ISSUE TITLE
Openvino output result is different from tensorflow after converting efficientnet-b8

### ISSUE THREAD

**Issue by TonyNgcn** [openvinotoolkit/openvino#2376]
##### System information (version)
- OpenVINO => 2020.4
- Operating System / Platform => Windows 10 64 Bit 19041
- Compiler => Visual Studio 2019
- Problem classification: Model Conversion
- Framework: tf-nightly  2.3.0.dev20200512
- Model name: efficientnet-b8

Problem:
I download the pre-trained model of efficientnet-b8 from https://storage.googleapis.com/cloud-tpu-heckpoints/efficientnet/advprop/efficientnet-b8.tar.gz  and git the code from https://github.com/tensorflow/tpu/tree/master/models/official/efficientnet
Then I use the export_model.py script to convert the ckpt file to SaveModel format. Then, I use the command "C:\Program Files (x86)\IntelSWTools\openvino_2020.4.287\deployment_tools\model_optimizer>python  mo_tf.py --saved_model_dir E:\model_downloads\tpu\models\official\efficientnet\efficientnet-b8\saved_model" to convert the model into IR file.

Finally, I found the infer result is different between two format of the model.
Test pic download from https://upload.wikimedia.org/wikipedia/commons/f/fe/Giant_Panda_in_Beijing_Zoo_1.JPG
The tensorflow script is
# -*- coding: utf-8 -*-
import eval_ckpt_main as eval_ckpt
import tensorflow.compat.v1 as tf
import sys

sys.path.append('/official/efficientnet')
sys.path.append('/common')
labels_map_file = 'E:\\model_downloads\\tpu\\models\\official\\efficientnet\\labels_map.json'
image_file = 'E:\\cocoDataset\\panda.jpg'
model_name = 'efficientnet-b8'
ckpt_dir = '.\\' + model_name

image_files = [image_file]
eval_driver = eval_ckpt.get_eval_driver(model_name)
pred_idx, pred_prob = eval_driver.eval_example_images(ckpt_dir, image_files, labels_map_file)

and the result is 
predicted class for image E:\cocoDataset\panda.jpg: 
  -> top_0 (81.84%): giant panda, panda, panda bear, coon bear, Ailuropoda melanoleuca  
  -> top_1 (1.23%): Arctic fox, white fox, Alopex lagopus  
  -> top_2 (0.94%): American black bear, black bear, Ursus americanus, Euarctos americanus  
  -> top_3 (0.36%): lesser panda, red panda, panda, bear cat, cat bear, Ailurus fulgens  
  -> top_4 (0.34%): brown bear, bruin, Ursus arctos  

The openvino script is on IntelSWTools\openvino_2020.4.287\inference_engine\samples\python\classification_sample
by using command "C:\Program Files (x86)\IntelSWTools\openvino_2020.4.287\inference_engine\samples\python\classification_sample>python cla
ssification_sample.py -m E:\model_downloads\efficientnet-b8\saved_model.xml -i E:\cocoDataset\panda.jpg --labels E:\mode
l_downloads\labels_map.txt"

The result is 

classid probability
------- -----------
122: 'American lobster, Northern lobster, Maine lobster, Homarus americanus',0.6933420
123: 'spiny lobster, langouste, rock lobster, crawfish, crayfish, sea crawfish',0.2900690
124: 'crayfish, crawfish, crawdad, crawdaddy',0.0048939
121: 'king crab, Alaska crab, Alaskan king crab, Alaska king crab, Paralithodes camtschatica',0.0014830
183: 'Kerry blue terrier',0.0003194
595: 'harvester, reaper',0.0003170
717: 'pickup, pickup truck',0.0002633
118: 'Dungeness crab, Cancer magister',0.0002430
491: 'chain saw, chainsaw',0.0002355
844: 'switch, electric switch, electrical switch',0.0001771

Can anyone solve my problem?

**Comment by jgespino** [comment]
Hi @TonyNgcn 

It's possible something may not have been converted correctly to SavedModel format or it may be your TensorFlow version. I was able to convert the model to IR format using the following commands and using the classification_sample obtained similar results to your TensorFlow output. I am using OpenVINO toolkit 2020.4 release with TensorFlow 1.14.

Please give this a try and let me know if it works for you.

Note: Commands below are for Linux but should be very similar in Windows. 
```
python3 /opt/intel/openvino/deployment_tools/model_optimizer/mo.py --input_meta model.ckpt.meta --output Softmax --input "IteratorGetNext:0[1 672 672 3]"
python3 classification_sample.py --model model.ckpt.xml --input panda.jpg --labels labels.txt
```
classid probability
------- -----------
388: 'giant panda, panda, panda bear, coon bear, Ailuropoda melanoleuca',0.8630295
279: 'Arctic fox, white fox, Alopex lagopus',0.0060065
295: 'American black bear, black bear, Ursus americanus, Euarctos americanus',0.0040638
425: 'barn',0.0028411
387: 'lesser panda, red panda, panda, bear cat, cat bear, Ailurus fulgens',0.0025269
294: 'brown bear, bruin, Ursus arctos',0.0022015
805: 'soccer ball',0.0017223
296: 'ice bear, polar bear, Ursus Maritimus, Thalarctos maritimus',0.0014338
971: 'bubble',0.0013110
270: 'white wolf, Arctic wolf, Canis lupus tundrarum',0.0012856

Regards,
Jesus

**Comment by jgespino** [comment]
Hi @TonyNgcn 

Did my previous suggestion work for you?

Regards,
Jesus
